rpc/src/lib.rs

```diff
@@ -75,7 +75,11 @@ impl RpcServer {
 	/// Start http server asynchronously and returns result with `Server` handle on success or an error.
 	pub fn start_http(&self, addr: &SocketAddr, cors_domains: Vec<String>) -> Result<Server, RpcServerError> {
 		let cors_domains = cors_domains.into_iter()
-			.map(jsonrpc_http_server::AccessControlAllowOrigin::Value)
+			.map(|v| match v {
+				ref v if v == "*" => jsonrpc_http_server::AccessControlAllowOrigin::Any,
+				ref v if v == "null" => jsonrpc_http_server::AccessControlAllowOrigin::Null,
+				v => jsonrpc_http_server::AccessControlAllowOrigin::Value(v),
+			})
```

Comment:
But then in case of `::Value` I need to copy it back to `String`. If you think it would be more readable this way I can change it.